2 - RTB/Verbali/Interni/23-11-13/23-11-13.typ

```diff
@@ -0,0 +1,84 @@
+#import "template.typ": *
+
+// Take a look at the file `template.typ` in the file panel
+// to customize this template and discover how it works.
+#show: project.with(
+  title: "Verbale **13/11/23**",
+  subTitle: "***Subtitle***",
+  docType: "verbale",        
+  partecipants: (
+    (name: "Banzato Alessio", role: "Verificatore"),
+    (name: "Carraro Riccardo", role: "Verificatore"),
+    (name: "Gardin Giovanni",  role: "Redattore"),
+    (name: "Nardo Silvio",   role: "Redattore"),
+    (name: "Oseliero Antonio",   role: "Amministratore"),
+    (name: "Todesco Mattia",  role: "Verificatore"),
+    (name: "Zaccone Rosario",  role: "Amministratore"),
+    //(name: "Pippo",  role: "Proponente"),
+  ),
+  oraInizio: "15:30",
+  oraFine:  "17:30",
+  firmeEsterne: "///inserire firma esterna///"   //Questa variabile va aggiunta solo se è un documento esterno, qua i partecipanti esterni andranno a firmarsi.
+)
+
+#set text(font:"New Computer Modern", size: 1.3em)
+#set align(left)
+
+
+
+= Ordine del giorno
+  - Aggiornamento del gruppo in merito alle nuove automazioni adottate;
+  - Discussione in merito al precedente sprint;
+  - Discussione in merito alle tecnologie adottate;
+  - Ripartizione dei ruoli per il nuovo sprint.
+
+= Automazioni e nuova repo
+  Il gruppo che si è dedicato nel precedente sprint ad automatizzare la repo ha 
+  finalizzato il lavoro e ha aggiornato il gruppo riguardo alle nuove funzionalità.
+  La vecchia repo è stata abbandonata in favore di una nuova (questo perchè riorganizzare 
+  la vecchia repo avrebbe richiesto troppo tempo) con una nuova organizzazione dei branch, 
+  differente rispetto alla precedente. La differenza fondamentale risiede nel fatto che prima si aveva 
+  un branch main per le release, un branch develop comune da cui si aprivano poi i vari branch feature 
+  di lavoro; ora abbiamo un branch main che racchiude tutti i pdf dei file, un branch orfano website 
```

Comment:
Non usare forme personali. Al posto di "abbiamo", utilizza forme come "ora il team possiede", "ora la repository è composta da".

Reply:
riformulerei la descrizione dei branch: 
"Ora la repository è composta da tre branch principali quali: 
- main: contenente i pdf compilati e pubblicati automaticamente tramite github actions solo a seguito della verifica di uno o più reviwer;
- src: contenente i file sorgenti scritti in Typst;
- website: branch parallello contente un versione work in progress del sito del team"

Reply:
il fatto di usare forme impersonali nella documentazione lo aggiungo anche a norme di progetto, era una delle domande che volevo farvi